lib/rules/no-yield-block-params-to-else-inverse.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const Rule = require('./base');
+// const AstNodeInfo = require('../helpers/ast-node-info');
+
+const ERROR_MESSAGE = 'Yielding block params to else/inverse block is not allowed';
+
+module.exports = class NoYieldBlockParamsToElseInverse extends Rule {
+  visitor() {
+    return {
+      MustacheStatement(node) {
+        if (node.path.original === 'yield') {
+          let isElseInverseHashPair = node.hash.pairs.some((p) => {
+            return p.key === 'to' && ['else', 'inverse'].includes(p.value.value);
```

Comment:
Reading `.value.value` will only work if the value is a static value and will error if we ever find a value that is **not** a static value.

Reply:
Can you please give some examples?

Reply:
`{{yield to=this.someValue}}`
`{{yield to=(get some this.map)}}`

Reply:
Thanks @lifeart

Reply:
See https://github.com/ember-template-lint/ember-template-lint/pull/1356/files#r436414696

Reply:
I think this still needs to be addressed.